### PR TITLE
Upload project

### PR DESCRIPTION
**Pull request by GreetyCr**
upload project for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,132 @@
...
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
```
Comment (GreetyCr) [inline README.md:22]: I see that you added a Installation section, if this is not needed, I suggest you kindly remove it to make it look more professional

File: style.css [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
+  -webkit-font-feature-settings: "liga";
+  -webkit-font-smoothing: antialiased;
+}
```
Comment (GreetyCr) [inline style.css:68]: [OPTIONAL] Hello, I noticed that your CSS is well-structured, but you might find it beneficial to consider the BEM (Block, Element, Modifier) methodology. It's a popular naming convention that can make your CSS more readable and easier to maintain. It's just a suggestion, though, and your current approach is still great. Keep up the good work!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!--CSS STYLES-->
+  <link rel="stylesheet" href="style.css">
+  <!-- Material Icons -->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <title>Youtube Replica</title>
+</head>
+<body>
+  <div class="wrapper">
```
Comment (GreetyCr) [inline index.html:14]: I suggest using semantic HTML for better SEO and best practices, I suggest you to change div's for semantic tags, more information here: https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md

**Review by GreetyCr — COMMENTED** [review]
Hi @GreetyCr,

Alt Text

### Don't Worry 🙏

It's ok that the project is not perfect as you first project is done all by yourself, but there are some aspects that we can improve before hoping into the next project

### Highlights

- ✔️ Nice overall layout.
- ✔️ Great design 
- ✔️ Correct GitHub flow

### Required Changes ♻️

Check the inline comments for more detailed information about the changes

### Optional suggestions ✅

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

### Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> Note: Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

### Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My GitHub: @GreetyCr 
- This is My Slack: Randall Bonilla
- This is My gmail: greetysoftwarejs@gmail.com

**Review by Ofarouq310 — APPROVED** [review]
Hi @GreetyCr, ✋ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇  

giphy

## Highlights
✅ Instructions followed properly.

## Optional suggestions

- It would be better if you incorporate the `checkbox` within your inline comments, that way when a student finishes one of the requirements they tick the checkbox to track what is left.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**